src/hepattn/utils/sorting.py

```diff
@@ -0,0 +1,127 @@
+import torch
+from torch import Tensor, nn
+
+
+class Sorter:
```

Comment:
If this has modules as members I think it should probably inherit from `nn.Module` itself